service/src/main/java/uk/gov/hmcts/reform/fpl/service/DocumentSealingService.java

```diff
@@ -32,12 +35,22 @@
 
     private final DocumentDownloadService documentDownloadService;
     private final UploadDocumentService uploadDocumentService;
+    private final DocumentConversionService documentConversionService;
 
-    public DocumentReference sealDocument(DocumentReference document) throws Exception {
-        byte[] documentContent = documentDownloadService.downloadDocument(document.getBinaryUrl());
-        byte[] sealedDocument = sealDocument(documentContent);
+    public DocumentReference sealAndUploadDocument(DocumentReference document) throws Exception {
```

Comment:
i think sealDocument(Document reference) is better. 
as an input user gives link to remote document, so expectation is that remote document will be sealed, no need to express it in name, client of this method does not care if document is download and then uploaded. Client cares only that remote doc is sealed